package.json

```diff
@@ -11,7 +11,7 @@
     "pretty-quick": "pretty-quick",
     "publish": "node tools/npm-publish.js",
     "test": "lerna run test --stream && npm run coverage",
-    "updateTestArtifacts": "node tools/updateTestArtifacts.js"
+    "updateTestArtifacts": "bash tools/updateTestSchema.sh"
```

Comment:
It would probably make sense to replace the bash script with a ShellJS/Shx wrapper, to keep it cross-platform

Reply:
hm, interesting idea.  However, the `updateTestSchema.sh` file was written so it can be used in every SDK regardless of the language.  I'm not sure it makes sense to write a ShellJS script when the other SDKs don't have a need for node or js.  Don't we use a [plethora of bash scripts](https://git.corp.adobe.com/TnT/m2/tree/develop/scripts) to make m2 work today?  I just assumed any developer running Windows would also have the Linux Bash Shell installed as well.